test/blackbox-tests/test-cases/pkg/just-print-solver-env.t

```diff
@@ -30,28 +27,27 @@ Add some build contexts with different environments
   >   (name no-doc)
   >   (lock dune.linux.lock)
   >   (solver_env
-  >    (flags (:standard \ with-doc)))))
+  >    (sys
```

Comment:
It's a bit strange that this appears here now, given only the flags were removed?

Reply:
This test exercises the code for printing the solver environment and it previously did this by setting some flags in the environment and then printing them. With the removal of flags this test would be left with an empty environment so I set some other variables so the printing code still gets tested.